### ISSUE TITLE
sudo make install doesnt install for 2.0.11, works fine for 2.0.10

### ISSUE THREAD

**Issue by dmsherazi** [eclipse/mosquitto#2234]
I am building from source , when I run make install in 2.0.11 I get the follwing result and mosquitto isnt installed.
The same for 2.0.10 works fine . 

```
sudo make install
set -e; for d in lib apps client plugins src; do make -C ${d}; done
make[1]: Entering directory '/home/myUser/development/mosquitto-2.0.11/lib'
make -C cpp
make[2]: Entering directory '/home/myUser/development/mosquitto-2.0.11/lib/cpp'
make[2]: Nothing to be done for 'all'.
make[2]: Leaving directory '/home/myUser/development/mosquitto-2.0.11/lib/cpp'
make[1]: Leaving directory '/home/myUser/development/mosquitto-2.0.11/lib'
make[1]: Entering directory '/home/myUser/development/mosquitto-2.0.11/apps'
set -e; for d in db_dump mosquitto_ctrl mosquitto_passwd; do make -C ${d}; done
make[2]: Entering directory '/home/myUser/development/mosquitto-2.0.11/apps/db_dump'
make[2]: Nothing to be done for 'all'.
make[2]: Leaving directory '/home/myUser/development/mosquitto-2.0.11/apps/db_dump'
make[2]: Entering directory '/home/myUser/development/mosquitto-2.0.11/apps/mosquitto_ctrl'
make[2]: Nothing to be done for 'all'.
make[2]: Leaving directory '/home/myUser/development/mosquitto-2.0.11/apps/mosquitto_ctrl'
make[2]: Entering directory '/home/myUser/development/mosquitto-2.0.11/apps/mosquitto_passwd'
make[2]: Nothing to be done for 'all'.
make[2]: Leaving directory '/home/myUser/development/mosquitto-2.0.11/apps/mosquitto_passwd'
make[1]: Leaving directory '/home/myUser/development/mosquitto-2.0.11/apps'
make[1]: Entering directory '/home/myUser/development/mosquitto-2.0.11/client'
make[1]: Nothing to be done for 'all'.
make[1]: Leaving directory '/home/myUser/development/mosquitto-2.0.11/client'
make[1]: Entering directory '/home/myUser/development/mosquitto-2.0.11/plugins'
set -e; for d in auth-by-ip dynamic-security message-timestamp payload-modification; do make -C ${d}; done
make[2]: Entering directory '/home/myUser/development/mosquitto-2.0.11/plugins/auth-by-ip'
make[2]: Nothing to be done for 'all'.
make[2]: Leaving directory '/home/myUser/development/mosquitto-2.0.11/plugins/auth-by-ip'
make[2]: Entering directory '/home/myUser/development/mosquitto-2.0.11/plugins/dynamic-security'
make[2]: Nothing to be done for 'all'.
make[2]: Leaving directory '/home/myUser/development/mosquitto-2.0.11/plugins/dynamic-security'
make[2]: Entering directory '/home/myUser/development/mosquitto-2.0.11/plugins/message-timestamp'
make[2]: Nothing to be done for 'all'.
make[2]: Leaving directory '/home/myUser/development/mosquitto-2.0.11/plugins/message-timestamp'
make[2]: Entering directory '/home/myUser/development/mosquitto-2.0.11/plugins/payload-modification'
make[2]: Nothing to be done for 'all'.
make[2]: Leaving directory '/home/myUser/development/mosquitto-2.0.11/plugins/payload-modification'
make[1]: Leaving directory '/home/myUser/development/mosquitto-2.0.11/plugins'
make[1]: Entering directory '/home/myUser/development/mosquitto-2.0.11/src'
make[1]: Nothing to be done for 'all'.
make[1]: Leaving directory '/home/myUser/development/mosquitto-2.0.11/src'
set -e; for d in man; do make -C ${d}; done
make[1]: Entering directory '/home/myUser/development/mosquitto-2.0.11/man'
xsltproc --nonet libmosquitto.3.xml
make[1]: xsltproc: Command not found
make[1]: *** [Makefile:90: libmosquitto.3] Error 127
make[1]: Leaving directory '/home/myUser/development/mosquitto-2.0.11/man'
make: *** [Makefile:57: docs] Error 2

```

**Comment by ptjm** [comment]
You need xsltproc and docbook-xsl installed to build the man pages. You can skip this step by putting WITH_DOCS=no as an argument to make. This does appear to be a mistake in the tar-building process since these dependencies are listed only "when building from git". The pre-built man pages were included with 2.0.9, but not with 2.0.10 or 2.0.11.

If you need the man pages but it's not convenient to install xsltproc, you could copy them over from your previous build. There's some new material in mosquitto.8, and mosquitto.conf.5, but the others haven't had appreciable changes recently.

**Comment by ralight** [comment]
I think this has been answered fine, so I'm closing the issue. If you have more questions, please feel free to ask.
